RevenueCatUI/Templates/LinTemplates/LinTemplateStep1View.swift

```diff
@@ -9,10 +9,45 @@ import Foundation
 import RevenueCat
 import SwiftUI
 
+struct LinTemplateStep1Configuration: Decodable {
+    enum CodingKeys: String, CodingKey {
+        case titleKey = "title_key"
+        case imageName = "image_name"
+    }
+    
+    let titleKey: String?
```

Comment:
I'm not super familiar with requirements - why is it optional? Don't we need to keep the old behaviour & set the proper one for a default value?

Reply:
I don't like it either, however there will be no title for a general case, just an image.